registry/storage/driver/s3-aws/s3.go

```diff
@@ -587,11 +587,20 @@ func (d *driver) Writer(ctx context.Context, path string, append bool) (storaged
 		if err != nil {
 			return nil, parseError(path, err)
 		}
-		var multiSize int64
```

Comment:
This multiSize was never used so is redundant code.  The size is calculated by the calling function